components/modal/src/css/popup-modal.css

```diff
@@ -0,0 +1,112 @@
+@import url(../../../../css/component.css);
+
```

Comment:
as you are import file which is in same folder better use url(./component.css) rather than going down from bottom.

Reply:
Its a duplicate in the folder which will be deleted later on.
Hence I have used this.
Thanks for suggestion.